hacks/ks-verify-helm.rb

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env ruby
+
+# verify charts specified in a helm repository render with ks
+# usage: ks-verify-helm.rb [helm repo URL]
+
+require 'net/http'
+require 'open3'
+require 'timeout'
+require 'tmpdir'
+require 'uri'
+require 'yaml'
+
+DEFAULT_HELM_REPO_URL="https://kubernetes-charts.storage.googleapis.com/index.yaml"
+
+helm_repo_url = ARGV[0] || DEFAULT_HELM_REPO_URL
+
+tmp_dir = Dir.mktmpdir("ks-verify-helm-", "/tmp")
```

Comment:
You might use `tmp_dir = Dir.mktmpdir("ks-verify-helm-", Dir.tmpdir)`

`Dir.tmpdir` on my macOS does not resolve to `/tmp`.

Reply:
It does, but I should use `Dir.tmpdir` if I ever wanted this to work in windows.

Reply:
Mine resolves to something like this:

```
/var/folders/gl/9th4vs815bv_bwgkxs9rmgr40000gn/T/ks-verify-helm-20180710-11142-19aege7
```